model/project.go

```diff
@@ -396,10 +396,11 @@ type PluginCommandConf struct {
 	// TimeoutSecs indicates the maximum duration the command is allowed to run for.
 	TimeoutSecs int `yaml:"timeout_secs,omitempty" bson:"timeout_secs,omitempty"`
 
-	// Params are used to supply configuration specific information.
-	Params map[string]interface{} `yaml:"params,omitempty" bson:"params,omitempty"`
+	// Params is used to define params in the yaml and parser project,
+	// but is not stored in the DB (instead see ParamsYAML).
+	Params map[string]interface{} `yaml:"params,omitempty" bson:"-"`
```

Comment:
Is this the only change needed to stop storing it in the dB?

Reply:
Yep! The struct tag "-" tells Go to ignore that field when parsing. Because we only change this for BSON, it ensures that we still read it in for yaml. This is also helpful for fields that we want to pass around within the code but don't want to actually store because it's repeated (for example, DisplayTask in the Task object). 
https://vsupalov.com/go-json-omitempty/#ignore-json-fields-when-parsing-it-even-though-your-struct-has-them